planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp

```diff
@@ -70,7 +70,9 @@ class NormalLaneChange : public LaneChangeBase
 
   void extendOutputDrivableArea(BehaviorModuleOutput & output) const override;
 
-  void insertStopPoint(const lanelet::ConstLanelets & lanelets, PathWithLaneId & path) override;
+  void insert_stop_point(const lanelet::ConstLanelets & lanelets, PathWithLaneId & path) override;
+
+  void insert_stop_point_on_current_lanes(PathWithLaneId & path);
```

Comment:
can these functions be `const` ?

Reply:
No, the function modifies member variable